lib/Connection.js

```diff
@@ -29,6 +29,13 @@ function Connection(router, req, res) {
         if (options) {
             if (options.props) request.setProps(options.props);
             if (options.resource) request.setResource(options.resource);
+
+            if (options.query) {
+                for (var key in options.query) {
```

Comment:
No, this is correct. The interface here is `connection#get(resourceId[, options])`. In Monocle clients, that will be translated to an HTTP request with stringified query string parameters:

    connection.get('/users', {
      query: {
        limit: 10,
        offset: 0
      }
    })

Which would be converted to an HTTP GET:

    GET /users?limit=10&offset=0

This is similar to how he handle `props`, which is passed in thusly:

    connection.get('/users/123', {
      props: ['name', 'age']
    })

Which could be converted to an HTTP GET:

    GET /users/123?props=name,age

The reason for this is to allow the resource ID to be used as a cache key and to make subsequent calls:

    monocleClient.get('/me')
    .then((user) => {
      user.name = 'Sally';
      // user.$resourceId may not be "/me", but the canonical path "/users/123"
      monocleClient.patch(user.$resourceId, {
       resource: user
      }).then((user) => {
        console.log('updated user!', user);
      });
    });